rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -231,45 +364,97 @@ s32 cellCameraGetType(s32 dev_num, vm::ptr<s32> type)
 	return CELL_OK;
 }
 
+s32 cellCameraGetAttribute(s32 dev_num, s32 attrib, vm::ptr<u32> arg1, vm::ptr<u32> arg2);
```

Comment:
Whats this for?

Reply:
`cellCameraIsAvailable` was calling it and it was declared below so I had to add the prototype.
Then I removed the code in the former that was calling it and the prototype remained up there.
I removed it because it was fetching `CELL_CAMERA_DEVICESPEED` which isn't written to in the current hle impl, but I just reversed it again and it's only being fetched if the camera is UVC, which means it wouldn't break the current impl. as I thought it would, so I'll add that back in correctly.

Reply:
`cellCameraIsAvailable` was calling it and it was declared below so I had to add the prototype.
Then I removed the code in the former that was calling it and the prototype remained up there.
I removed it because it was fetching `CELL_CAMERA_DEVICESPEED` which isn't written to in the current hle impl. I'm not even sure what it's supposed to represent so I didn't try to hle it more properly.
I'll remove the prototype.